tools/windows-node-installer/test/e2e/azure_test.go

```diff
@@ -215,6 +224,19 @@ func areRequiredRulesPresent(secGroupRules []network.SecurityRule) bool {
 	return true
 }
 
+// getIpPass extracts instance IPAddress and password from instance rdp file.
+func getIpPass(vmName string) (string, string, error) {
+	vmCredentialPath := filepath.Join(dir, "/", vmName)
+	rdpCmd, err := ioutil.ReadFile(vmCredentialPath)
+	if err != nil {
+		return "", "", fmt.Errorf("failed to read %s credentials with error: %v", vmName, err)
+	}
+	ipAddress := ipAddressPattern.FindString(string(rdpCmd))
+	password := passwordPattern.FindString(string(rdpCmd))[3:]
```

Comment:
Why are you dropping the first few characters. You need to have a comment explaining this